### PR TITLE
Switch to Renesas Yocto-3.9

### PR DESCRIPTION
**Pull request by iartemenko**
Set proper commit hash in order to use Renesas Yocto-3.9.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: prod_gen3_test/domd.xml [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
     <default remote="github" sync-j="10" sync-c="true" />
 
-    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="d50e055494cea5de0a348c15ee27cb67a3542e0b" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="fd078b6ece537d986852cb827bd21e022a797b2f" />
     <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="rocko" revision="rocko" />
     <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="342fbd6a3e57021c8e28b124b3adb241936f3d9d" />
     <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="dacfa2b1920e285531bec55cd2f08743390aaf57" />
     <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
     <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="morty" />
+    <project remote="epam" name="epmd-aepr/img-proprietary" path="proprietary" upstream="master" revision="master" />
```
Comment (otyshchenko1) [inline prod_gen3_test/domd.xml]: Why did you add this line?

File: prod_gen3_test/domd.xml [hunk 2]
```diff
@@ -4,13 +4,15 @@
     <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
     <remote name="openembedded" fetch="git://git.openembedded.org" />
     <remote name="linaro" fetch="git://git.linaro.org" />
+    <remote name="epam" fetch="ssh://git@gitpct.epam.com" />
```
Comment (otyshchenko1) [inline prod_gen3_test/domd.xml]: Why did you add this line?

**Comment by otyshchenko1** [comment]
Please drop Gen3-test from this PR. I will create separate PR.

**Comment by iartemenko** [comment]
Done
